src/aind_metadata_extractor/models/fip_legacy.py

```diff
@@ -0,0 +1,43 @@
+"""Data models of extracted information from FIP legacy files."""
+
+from typing import Literal, Union, Optional, List
+from datetime import datetime
+from pathlib import Path
+from pydantic import BaseModel, Field
+
+
+class FiberData(BaseModel):
+    """
+    Intermediate data model for fiber photometry data.
+
+    This model holds the extracted and processed data before final
+    transformation into a Session object. It serves as a structured
+    intermediate representation of the fiber photometry session data.
+    """
+
+    job_settings_name: Literal["FiberPhotometry"] = Field(
+        default="FiberPhotometry", description="Name of the job settings type"
+    )
+    experimenter_full_name: List[str] = Field(..., description="List of experimenter names")
+    session_start_time: Optional[datetime] = Field(None, description="Start time of the session")
```

Comment:
We should be consistent with the use of 'default=...'. Either always use it or not.